docs/ossm/kiali-multi-cluster/README.md

```diff
@@ -0,0 +1,137 @@
+     namespace: istio-system
+   spec:
+     version: v1.89
+   ```
+
+   Apply the yaml file into the `east` cluster.
+
+   ```sh
+   oc --context east apply -f kiali.yaml
+   ```
+
+   Wait for Kiali to become ready.
+
+   ```sh
+   oc --context east wait --for=condition=Successful --timeout=60s kialis/kiali -n istio-system
+   ```
+
+1. Create an `OAuthClient` on the remote cluster so that Kiali can access the OpenShift API server on behalf of users.
+
+   Find your Kiali URL
+
+   ```sh
+   oc --context east get route kiali -n istio-system -o jsonpath='{.spec.host}'
+   ```
+
+   Create a file named `oauthclientwest.yaml`
+
+   ```yaml
+   apiVersion: oauth.openshift.io/v1
+   grantMethod: auto
+   kind: OAuthClient
+   metadata:
+     labels:
+       app: kiali
+       app.kubernetes.io/instance: kiali
+       app.kubernetes.io/name: kiali
+       app.kubernetes.io/part-of: kiali
+     name: kiali-istio-system
+   redirectURIs:
+     - https://<your-kiali-route>/api/auth/callback/west
+   ```
+
+   Create the `OAuthClient` in the west cluster.
+
+   ```sh
+   oc --context west apply -f oauthclientwest.yaml
+   ```
+
+1. Create a remote cluster secret.
+
+   In order to access a remote cluster, you must provide a kubeconfig to Kiali via a Kubernetes secret. You can use [this script](https://raw.githubusercontent.com/kiali/kiali/master/hack/istio/multicluster/kiali-prepare-remote-cluster.sh) to simplify this process for you. Running this script will:
+
+   - Create a Service Account for Kiali in the remote cluster.
+   - Create RBAC resources for this Service Account in the remote cluster.
+   - Create a kubeconfig file and save this as a secret in the namespace where Kiali is deployed on the `east` cluster.
+
+   1. Download the `kiali-prepare-remote-cluster` script.
+
+      ```sh
+      curl -L -o kiali-prepare-remote-cluster.sh https://raw.githubusercontent.com/kiali/kiali/master/hack/istio/multicluster/kiali-prepare-remote-cluster.sh
+      ```
+
+   2. Make the script executeable.
+
+      ```sh
+      chmod +x kiali-prepare-remote-cluster.sh
+      ```
+
+   3. Run the script passing your `east` and `west` cluster contexts.
+
+      ```sh
+      ./kiali-prepare-remote-cluster.sh --kiali-cluster-context east --remote-cluster-context west --view-only false --kiali-resource-name kiali --remote-cluster-namespace istio-system --remote-cluster-name west
+      ```
+
+   **Note:** Use the option `--help` for additional details on how to use the script.
+
+1. Restart Kiali to pickup the remote secret.
+
+   ```sh
+   oc --context east rollout restart deployments/kiali -n istio-system
+   ```
+
+   Wait for Kiali to become ready
+
+   ```sh
+   oc --context east rollout status deployments/kiali -n istio-system
+   ```
+
+1. Login to Kiali.
+
+   When you first visit Kiali, you will login to the cluster where Kiali is deployed. In our case it will be the `east` cluster.
+
+   ```sh
+   oc --context east get route -l app=kiali -n istio-system
```

Comment:
Kinda confused what this command was for. I think you mean for this to show where the user should point his browser. Perhaps something more specific - this opens a browser tab right to Kiali, maybe use this:

```
xdg-open "https://$(oc --context east get route -l app=kiali -n istio-system -o jsonpath='{..spec.host}')"
```

Reply:
Oh, also, I try not to use "app" label in my official docs because technically we should be telling people to use the `app.kubernetes.io/` labels ("app" is unofficially deprecated - we want to get people to use the standard names). So..

```
xdg-open "https://$(oc --context east get route -l app.kubernets.io/name=kiali -n istio-system -o jsonpath='{..spec.host}')"
```

Reply:
I may just say "open your browser to..." here to be more platform agnostic.